contact_updater/views.py

```diff
@@ -0,0 +1,107 @@
+import json
+import time
+
+from contact_updater.forms import AgencyData
+
+from django.forms.formsets import formset_factory
+from django.shortcuts import render
+from django.http import HttpResponse
+
+from foia_hub.api import AgencyResource, OfficeResource
+
+
+def form_index(request):
+    agencies = AgencyResource().list()
+    return render(
+        request, "form_index.html", {'agencies': agencies})
+
+
+def download_data(request):
+    """ Converts POST request into json file ready for download """
+
+    data = dict(request.POST)
+    data['timestamp'] = int(time.time())
+    del data['csrfmiddlewaretoken']
+    res = HttpResponse(json.dumps(data), content_type="application/javascript")
+    res['Content-Disposition'] = 'attachment; filename=contact_data.json'
+    return res
+
+
+def get_first_element(array):
+    """ Given a list returns first element """
```

Comment:
I think having this function is weird, all you're doing is return the first element in a list. Can we remove the need for it?